server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java

```diff
@@ -64,7 +64,7 @@ public Boolean call() throws Exception {
 
   private Optional<CeTask> tryAndFindTaskToExecute() {
     try {
-      return queue.peek();
+      return queue.peek("UNKNOWN" /*FIXME provide a real worker uuid*/);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)